x-pack/test/apm_api_integration/trial/tests/transaction_groups/error_rate.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+import expectedErrorRate from './expectation/error_rate.json';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const supertestAsApmReadUser = getService('supertestAsApmReadUser');
+  const esArchiver = getService('esArchiver');
+
+  // url parameters
+  const start = encodeURIComponent('2020-06-29T06:45:00.000Z');
+  const end = encodeURIComponent('2020-06-29T06:49:00.000Z');
+  const uiFilters = encodeURIComponent(JSON.stringify({}));
+
+  describe('Error rate', () => {
+    describe('when data is not loaded', () => {
+      it('handles the empty state', async () => {
+        const response = await supertestAsApmReadUser.get(
+          `/api/apm/services/opbeans-node/transaction_groups/error_rate?start=${start}&end=${end}&uiFilters=${uiFilters}`
+        );
+        expect(response.status).to.be(200);
+        expect(response.body).to.eql({
+          noHits: true,
+          erroneousTransactionsRate: [],
+          average: null,
+        });
+      });
+    });
+    describe('when data is loaded', () => {
+      before(() => esArchiver.load('8.0.0'));
+      after(() => esArchiver.unload('8.0.0'));
+
+      it('returns the transaction error rate', async () => {
+        const response = await supertestAsApmReadUser.get(
+          `/api/apm/services/opbeans-node/transaction_groups/error_rate?start=${start}&end=${end}&uiFilters=${uiFilters}`
+        );
+
+        expect(response.status).to.be(200);
+        expect(response.body).to.eql(expectedErrorRate);
+      });
+    });
+  });
+}
```

Comment:
I thought I wrote this comment already but I can't find it so might have forgotten to submit: I don't think we should have duplicate `basic` and `trial` tests if they are exactly the same. As a rule of thumb we should test everything in basic, and only add `trial` tests if behaviour changes with the license.

Reply:
It would be great if we could annotate tests with a license, thereby making it possible to run the same test for both licenses without duplicating it.

Reply:
> I don't think we should have duplicate `basic` and `trial` tests if they are exactly the same

Thanks for clarifying it, I thought it was useless duplicating the test too. Just removed it.